site/docs/tools/iceberg/spark.md

```diff
@@ -204,8 +205,9 @@ addressing jvm objects directly in Python (until the python library for iceberg
     # first instantiate the catalog
```

Comment:
Spark-3.x currently says python code is the same as spark-2.x and doesn't have an example for that. 

I have the plan to drop spark-2.x mention in the document in a follow-up PR. That time I will move this example under spark-3.x